core/frontend/src/tile/map/MapLayerFormatRegistry.ts

```diff
@@ -10,25 +10,28 @@ import { assert } from "@itwin/core-bentley";
 import { ImageMapLayerSettings, MapLayerKey, MapLayerSettings, MapSubLayerProps } from "@itwin/core-common";
 import { IModelApp } from "../../IModelApp";
 import { IModelConnection } from "../../IModelConnection";
-import { ImageryMapLayerTreeReference, internalMapLayerImageryFormats, MapLayerAuthenticationInfo, MapLayerImageryProvider, MapLayerSourceStatus, MapLayerTileTreeReference } from "../internal";
+import { ImageryMapLayerTreeReference, internalMapLayerImageryFormats, MapLayerAccessClient, MapLayerAuthenticationInfo, MapLayerImageryProvider, MapLayerSourceStatus, MapLayerTileTreeReference } from "../internal";
 import { RequestBasicCredentials } from "../../request/Request";
 
-/** @internal */
+/** @alpha */
 export class MapLayerFormat {
```

Comment:
You mean an interface?  I guess it should be an interface yes. Although I think Ray wanted to have a default 'validateSource' behavior.  That could be move on `ImageryMapLayerFormat` instead.

Having an optional AccessClient on the `MapLayerFormat` interface class/interface would be another good way of implementing it...We would need to provided a way lookup a `MapLayerFormat` object from a formatId string on `MapLayerFormatRegistry` though.

Do you prefer that design rather than having a new `MapLayerFormatEntry` interface?